app/models/qset.rb

```diff
@@ -9,7 +9,8 @@ class Qset < ActiveRecord::Base
     s.key :permissions, :defaults => {
         :all_questions_visible => qset_defaults.all_questions_visible,
         :question_authors_visible => qset_defaults.question_authors_visible,
-        :questions_visible_to_unauth_user => qset_defaults.questions_visible_to_unauth_user
+        :questions_visible_to_unauth_user => qset_defaults.questions_visible_to_unauth_user,
+        :toggle_qsets => qset_defaults.toggle_qsets
```

Comment:
I find `toggle_qsets` a little opaque as a description of this setting/behavior. What about `qset_view` or `qset_type`, for instance? It's a string that could (theoretically) be one of many different values, so `toggle_qsets` doesn't seem quite right.